snek/exts/bookmark.py

```diff
@@ -0,0 +1,58 @@
+import discord
+from discord.ext.commands import Cog, Context, command
+
+from snek.bot import Snek
+
+
```

Comment:
Adding the thumbnail url constant
```suggestion
THUMBNAIL_URL = 'https://cdn.iconscout.com/icon/free/png-32/bookmark-1754138-1493251.png'


```